From 995aaf30799fa972441354b6feb45f0621968929 Mon Sep 17 00:00:00 2001
From: Jim Somerville <Jim.Somerville@windriver.com>
Date: Wed, 16 Oct 2013 16:41:03 -0400
Subject: [PATCH] Set up for cross compilation

Message-Id: <17e5a28667f667859c48bee25e575a072d39ee1b.1381956170.git.Jim.Somerville@windriver.com>

Set the host compiler to BUILD_CXX so
proper cross compilation can occur.

Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>

---
Upstream-Status: Pending

 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index fe80ee0..57cd1ac 100644
--- a/configure.ac
+++ b/configure.ac
@@ -27,6 +27,7 @@ m4_include([acinclude/lib-checks.m4])
 m4_include([acinclude/ax_cxx_compile_stdcxx_11.m4])
 m4_include([acinclude/ax_cxx_0x_types.m4])
 
+HOSTCXX="$BUILD_CXX"
 PRESET_CFLAGS="$CFLAGS"
 PRESET_CXXFLAGS="$CXXFLAGS"
 PRESET_LDFLAGS="$LDFLAGS"
